eng/pipelines/libraries/helix-queues-setup.yml

```diff
@@ -78,9 +78,7 @@ jobs:
       - ${{ if eq(parameters.jobParameters.isFullMatrix, true) }}:
         - OSX.1013.Amd64.Open
         - OSX.1014.Amd64.Open
-        # The 10.15 machines aren't in the same configuration, see
-        # https://github.com/dotnet/runtime/issues/24736
-        #- OSX.1015.Amd64.Open
+        - OSX.1015.Amd64.Open
```

Comment:
Should we do 1014 and 1015 on PRs and then the 3 on CI?

I feel like it is more important to have latest 2 on PRs. `isFullMatrix == false` is PRs.